### PR TITLE
[Sprint: 49] XD-3077 Set HDFS as custom module registry for YARN deployments

### PR DESCRIPTION
**Pull request by trisberg**
- Update docs

### CODE REVIEW

File: spring-xd-yarn/site/config/servers.yml [hunk 1]
```diff
@@ -63,6 +72,9 @@ spring:
 #  hadoop:
 #    fsUri: hdfs://localhost:8020
 #    resourceManagerHost: localhost
+#    resourceManagerPort: 8032
+#    resourceManagerSchedulerAddress: ${spring.hadoop.resourceManagerHost:8030}
+#    jobHistoryAddress: ${spring.hadoop.resourceManagerHost:10020}
```
Comment (jvalkeal) [inline spring-xd-yarn/site/config/servers.yml]: where is `spring.hadoop.jobHistoryAddress` used? We don't have that in shdp conf shortcuts.

Comment (trisberg) [inline spring-xd-yarn/site/config/servers.yml]: It's not used, it is needed for custom batch jobs so I though we should include it here

Comment (jvalkeal) [inline spring-xd-yarn/site/config/servers.yml]: ah, ok. I'll need to add that to shdp as we talked a while back.

**Comment by jvalkeal** [comment]
ok, looks good. I'll merge this in a bit.

**Comment by jvalkeal** [comment]
merged

**Pull request closed without merging**